src/sites/doc/docs/taro/contributing-react.en-US.md

```
@@ -0,0 +1,127 @@
+
+&nbsp;
+
+Before you commit a bug, Please ensure that the search had existing [issue] (https://github.com/jdf2e/nutui-react/issues) and read our [FAQ] (https://nutui.jd.com/#/zh-CN/notice).
+
+## new features
+
+If you have a better idea of an existing component function or API, We also recommend that you use our provided [issue - helper] (https://nutui.jd.com/nutui-issue-helper/?repo=jdf2e/nutui-react) to submit a issue of adding new features.
+
+&nbsp;
+
+If you want to help develop new features and components, Can scan [qr code] (HTTP: / / https://img13.360buyimg.com/imagetools/s300x300_jfs/t1/187308/9/29678/349561/634fa983E93012590/a641089fdfed5911 .png) join our community co-building group.
```

Markdownlint (0.37.0)

Bare URL used

(MD034, no-bare-urls)